spec/serializers/atmosphere/virtual_machine_flavor_serializer_spec.rb

```diff
@@ -1,38 +1,66 @@
 require 'rails_helper'
 
 describe Atmosphere::VirtualMachineFlavorSerializer do
-  it 'sets active flag to true for flavor on active compute site' do
-    flavor = flavor_on_cs(cs_active: true, active: true)
 
-    result = serialize(flavor)
+  context 'basic tests' do
+    it 'sets active flag to true for flavor on active compute site' do
+      flavor = flavor_on_cs(cs_active: true, active: true)
 
-    expect(result['virtual_machine_flavor']['active']).to be_truthy
-  end
+      result = serialize(flavor)
 
-  it 'sets active flag to true for flavor on inactive compute site' do
-    flavor = flavor_on_cs(cs_active: false, active: true)
+      expect(result['virtual_machine_flavor']['active']).to be_truthy
+    end
 
-    result = serialize(flavor)
+    it 'sets active flag to true for flavor on inactive compute site' do
```

Comment:
sets active flag to _true_ for flavor on inactive compute site => sets active flag to _false_ for flavor on inactive compute site